src/pid/PIDRunner.java

```diff
@@ -0,0 +1,7 @@
+package pid;
+
+public interface PIDRunner {
+  void startPIDLoop();
+
+  void stopPIDLoop(int remainOrStop);
```

Comment:
Should be a boolean, something like `shouldStop`

Reply:
this function is supposed to be in the MotorControllerEnhancedPIDController class no?
`

Reply:
@AE173 I think we didn't include that in the new implementation, make sure it's in.